Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.cpp

```diff
@@ -317,7 +317,7 @@ void RemoteDisplayListRecorder::drawNativeImageWithQualifiedIdentifier(Qualified
     handleItem(DisplayList::DrawNativeImage(imageIdentifier.object(), imageSize, destRect, srcRect, options), *image);
 }
 
-void RemoteDisplayListRecorder::drawSystemImage(SystemImage& systemImage, const FloatRect& destinationRect)
+void RemoteDisplayListRecorder::drawSystemImage(const Ref<SystemImage>& systemImage, const FloatRect& destinationRect)
```

Comment:
Needing to have `const Ref<T>` in the signature looks like a regression.
This could perhaps be worked around by doing an unrelated improvement described below..

Reply:
Ref<>&& is more appropriate in general, forgot it's possible.